packages/gatsby-theme-docs/gatsby-ssr.js

```diff
@@ -64,9 +59,9 @@ export const replaceRenderer = ({
     />,
     <style
       key="emotion-ssr"
-      data-emotion-css={ids.join(' ')}
+      data-emotion={`${docsCacheKey} ${ids.join(' ')}`}
```

Comment:
I saw this in the emotion repo examples/tests, so we're doing the same thing here.